core/os/device/remotessh/device.go

```diff
@@ -201,5 +201,11 @@ func getConnectedDevice(ctx context.Context, c Configuration) (Device, error) {
 
 	b.To = &device
 	b.To.Name = c.Name
+
+	for i := range b.To.ID.Data {
```

Comment:
As discussed offline, I'm not sure why we have different IDs for the same device here.